week2/assignment1-1/ex2.cpp

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
+#include <array>
+#include <algorithm>
+
+/* Add a function that receives an std::array and a number, 
+ * and checks whether the number is in the array. The function 
+ * should return a boolean value.*/
+
+bool is_number_in_array(std::array<int, 10> arr, int num) {
+    auto exists = std::find(begin(arr), end(arr), num);
+    return (exists != std::end(arr));
+}
+
+int main() {
+    std::array arr{3, 73, 5, 90, 100, 4, 88, 2, 5001, 62};
```

Comment:
When initializing arrays its good to give the size here. std::array<int, 10> arr. And here the name is not an issue, but a more descriptive name is better.